src/app/transactions/container.tsx

```diff
@@ -1,15 +1,41 @@
 import * as React from 'react'
+const { connect } = require('react-redux')
+import { bindActionCreators, Dispatch } from 'redux'
+import TransactionsComponent from '../common/transactions'
+import { getTransactions } from '../transactions/actions'
+import { DispatchProps, State, State as Props } from './types'
+
+// tslint:disable-next-line:typedef
+const mapStateToProps = (state: State): Props =>
+  ({
+    transactions: state.transactions
+  })
+
+// tslint:disable-next-line:typedef
+const mapDispatchToProps = (dispatch: Dispatch<State>): DispatchProps =>
+  bindActionCreators({
+    getTransactions
+    // tslint:disable-next-line:align
+  }, dispatch)
+
+@connect(mapStateToProps, mapDispatchToProps)
+export default class Transactions extends React.Component<Props & DispatchProps, any> {
+  // tslint:disable-next-line:typedef
+  public componentDidMount() {
+    this.props.getTransactions()
+  }
 
-export default class Transactions extends React.Component {
   public render(): any {
+    const transactions: any = this.props.transactions.transactions || []
```

Comment:
Do we need this ` || []`? The default state on the reducer is the empty array already right?